File: src/app/model/tictactoe/subscribed-user.ts
export class SubscribedUser {

    userName: string;
    sessionId: string;

    constructor() {
        this.userName = "";
        this.sessionId = "";
    }

    public toString(): string {
        return ("SubscribedUser [userName=" + this.userName + ", sessionId=" + this.sessionId + "]");
    }
}

File: src/main/java/rd/sb_ng_browsergames/model/dto/SubscribedUser.java
package rd.sb_ng_browsergames.model.dto;

public class SubscribedUser {

    private String userName;
    private String sessionId;

    
    public SubscribedUser(String userName, String sessionId) {
        this.userName = userName;
        this.sessionId = sessionId;
    }


    public String getUserName() {
        return userName;
    }

    public void setUserName(String userName) {
        this.userName = userName;
    }

    public String getSessionId() {
        return sessionId;
    }

    public void setSessionId(String sessionId) {
        this.sessionId = sessionId;
    }

    @Override
    public String toString() {
        return "SubscribedUser [userName=" + userName + ", sessionId=" + sessionId + "]";
    }

}
